script.js

```diff
@@ -1,70 +1,95 @@
-document.addEventListener('DOMContentLoaded', () => {
-  const bookForm = document.getElementById('add-book');
-  const list = document.querySelector('.book-list');
+/* eslint max-classes-per-file: ["error", 3] */
+// Book Class
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// Store Class: Handles Storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
 
-  let books = [];
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
 
-  // REMOVE BOOKS
-  function removeBook(id) {
-    books = books.filter((book) => {
-      if (book.id === id) {
-        return false;
+  static removeBook(author) {
+    const books = Store.getBooks();
+    books.forEach((book, index) => {
+      if (book.author === author) {
+        books.splice(index, 1);
       }
```

Comment:
- [ ] Great work writing the code for this function. However, I noticed that if I have an author with the same name and I delete one book, initially it deletes one book from the UI but when I reload, more than one book is deleted. There are two reasons for this;

1. You are looping through the `books` array and mutating the original array by using the `splice` method. This will make the removal of books not accurate. 😎 
2. You are using the condition of the `author` to remove a book. Think of it this way, in the real world, you can add several books with the same `author` or even `title`. Also, when you want to remove a book, you would want to remove a single book and not all the books with the same title or author. 😎 

#### Hint: To fix it, I would recommend you assign each book an `ID` when that is unique when creating them and then use the [filter](https://www.geeksforgeeks.org/how-to-filter-object-array-based-on-attributes/) method to remove the book if the `index` is equal to the `ID`. You don't have to do it this way if it isn't convenient for you, as long as you remove a unique book even if they have the same _**author**_ and **_title_**. 👍 

### Demo of Bug

![chrome-capture-2023-1-21](https://user-images.githubusercontent.com/77106962/220463240-7bc14cb6-2712-4f39-bc5a-361bb463fa73.gif)